_data/pubs.yml

```
@@ -67,7 +67,7 @@
   year: 2021
   pages: 
   links:
-     paper: "https://apertureneuropub.cloud68.co/articles/42/index.html"
+     paper: "https://apertureneuro.org/article/77474-brainiak-the-brain-imaging-analysis-kit"
```

What do you think about using the DOI link?
```suggestion
     paper: "https://doi.org/10.52294/31bb5b68-2184-411b-8c00-a1dacb61e1da"
```

---

Good idea!